### DISCUSSION TITLE
More than one audio file in one track using multitrack plugin?

### DISCUSSION THREAD

**Discussion by FSDRahul** [katspaugh/wavesurfer.js#3661] [Q&A]
Hi katspaugh,

I need help regarding a use case where I need to generate waveforms with the multitrack plugin.
How can i use the multitrack plugin  to support the below use case.

We are building a multitrack audio editor. In one track we have more than 1 audio files and its waveforms and they need to be created at different intervals.

Ex. shown below but we need waveforms 
![Screenshot 2024-04-16 123027](https://github.com/katspaugh/wavesurfer.js/assets/138694810/d2e7eb43-6720-456a-b4b6-f06085430d23)

**Comment by katspaugh** [comment]
You can probably start with those tracks stacked under each other as per the default multitrack behavior, and then add a negative margin to overlay them on the previous track.

Please note that I don’t offer free support for multitrack as stated in its readme.
